bot/src/bot/callback/request.callback.ts

```diff
@@ -0,0 +1,54 @@
+import {Callback} from "./callback";
+import TelegramBot from "node-telegram-bot-api";
+import {Injectable} from "@nestjs/common";
+import {FriendRequestService} from "../../friend-request/friend-request.service";
+import {BotService} from "../bot.service";
+
+@Injectable()
+export class RequestCallback extends Callback {
+  name = 'request'
+
+  constructor(private friendRequestService: FriendRequestService, private botService: BotService) {
+    super();
+  }
+
+  // TODO: Notify friend request sender
+  async handle(callbackQuery: TelegramBot.CallbackQuery): Promise<void> {
+    if(!callbackQuery.data)
+      return;
+
+    const data = callbackQuery.data.substring(this.name.length+1).split('_');
```

Comment:
put it in a separate function

Reply:
Let's do it when we'll refactor this methods